source/includes/pipeline_config/_018-ant-task.md.erb

```diff
@@ -0,0 +1,26 @@
+<%
+json ='{
+  "tasks": [
+    {
+      "type": "ant",
+      "attributes": {
+        "run_if": [
+          "passed"
+        ],
+        "working_directory": "script/build",
+        "build_file": null,
+        "target": null
+      }
+    }
+  ]
+}'
+%>
+<%=
+describe_sub_object 'The ant task attributes', json: json do
+  run_if                       'Array',        'The run_if condition specifies when a task should be allowed to run.', one_of: %w(passed failed any)
+  build_file                   'String',       'The path to Ant build file.'
+  target                       'String',       'The ant target(s) to run.'
```

Comment:
Yes, that makes sense to me. As long as the args are split on space (they should be, if there can be multiple targets as it stands today).